README.md

```diff
@@ -1,742 +1,77 @@
-
-### Exporters Packages
-
-| Package              | MyGet (CI)                                                                                                       | NuGet (releases)                                                                                                 |
-| -------------------- | ---------------------------------------------------------------------------------------------------------------- | ---------------------------------------------------------------------------------------------------------------- |
-| Zipkin               | [![MyGet Nightly][OpenTelemetry-exporter-zipkin-myget-image]][OpenTelemetry-exporter-zipkin-myget-url]           | [![NuGet release][OpenTelemetry-exporter-zipkin-nuget-image]][OpenTelemetry-exporter-zipkin-nuget-url]           |
-| Prometheus           | [![MyGet Nightly][OpenTelemetry-exporter-prom-myget-image]][OpenTelemetry-exporter-prom-myget-url]               | [![NuGet release][OpenTelemetry-exporter-prom-nuget-image]][OpenTelemetry-exporter-prom-nuget-url]               |
-| Application Insights | [![MyGet Nightly][OpenTelemetry-exporter-ai-myget-image]][OpenTelemetry-exporter-ai-myget-url]                   | [![NuGet release][OpenTelemetry-exporter-ai-nuget-image]][OpenTelemetry-exporter-ai-nuget-url]                   |
-| Stackdriver          | [![MyGet Nightly][OpenTelemetry-exporter-stackdriver-myget-image]][OpenTelemetry-exporter-stackdriver-myget-url] | [![NuGet release][OpenTelemetry-exporter-stackdriver-nuget-image]][OpenTelemetry-exporter-stackdriver-nuget-url] |
-| Jaeger               | [![MyGet Nightly][OpenTelemetry-exporter-jaeger-myget-image]][OpenTelemetry-exporter-jaeger-myget-url]           | [![NuGet release][OpenTelemetry-exporter-jaeger-nuget-image]][OpenTelemetry-exporter-jaeger-nuget-url]           |
-| LightStep            | [![MyGet Nightly][OpenTelemetry-exporter-lightstep-myget-image]][OpenTelemetry-exporter-lightstep-myget-url]     | [![NuGet release][OpenTelemetry-exporter-lightstep-nuget-image]][OpenTelemetry-exporter-lightstep-nuget-url]     |
-| Honeycomb            |                                                                                                                  | [![NuGet release][OpenTelemetry-exporter-honeycomb-nuget-image]][OpenTelemetry-exporter-honeycomb-nuget-url]     |
-| NewRelic             |                                                                                                                  | [![NuGet release][OpenTelemetry-exporter-newrelic-nuget-image]][OpenTelemetry-exporter-newrelic-nuget-url]       |
-| Console              | [![MyGet Nightly][OpenTelemetry-exporter-console-myget-image]][OpenTelemetry-exporter-console-myget-url]         |                                                                                                                  |
-
-
-## OpenTelemetry Tracing QuickStart: Collecting Data

```

Comment:
Can you move this into a separate document for now so it wouldn't be lost while we are in transition

Reply:
I've moved most of the stuff to specific folder.
The general document on tracing and metrics need to be rewritten and I'll address it in my next PR.

Reply:
there is also sections on writing custom sampler and exporter. Would you mind just copy it for now as a separate doc, shouldn't be a problem to remove it with the next PR. Just want to keep some docs around.

Reply:
Good idea, I've moved it to a separate doc under `/docs` folder.